app/controllers/hyrax/controller.rb

```diff
@@ -4,6 +4,7 @@ module Hyrax::Controller
   included do
     class_attribute :create_work_presenter_class
     self.create_work_presenter_class = Hyrax::SelectTypeListPresenter
+    self.search_state_class = Hyrax::SearchState
```

Comment:
But the local controller.rb is not identical to hyrax/controller.rb. It seems there are some local implementation in controller.rb. Are you sure it can be safely removed?